src/components/themed/WalletListErrorRow.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
+import { TouchableOpacity } from 'react-native'
+import EntypoIcon from 'react-native-vector-icons/Entypo'
+
+import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
+import { EdgeText } from './EdgeText'
+
+interface Props {
+  onLongPress?: () => void
+  error: Error
+}
+
+function WalletListErrorRowComponent(props: Props) {
+  const { onLongPress, error } = props
+  const theme = useTheme()
+  const styles = getStyles(theme)
+
+  return (
+    <TouchableOpacity style={styles.container} onLongPress={onLongPress}>
+      <EntypoIcon name="warning" size={theme.rem(1)} style={styles.icon} />
+      <EdgeText>{error.message}</EdgeText>
+    </TouchableOpacity>
```

Comment:
I think we upgrade the visuals later, such as adding the `CryptoIcon` for the wallet, but for now this is sooo much better.